<?xml version="1.0"?>
<launch>
    
    <node name="detector_2d_node" pkg="detector_2d" type="main_detect.py" output="screen">
        <rosparam command="load" file="$(find detector_2d)/config/main_cfg.yaml"/>
        <param name="image_topic" value="/usb_cam/image_raw" />
        <param name="point_cloud_topic" value="/camera/depth_registered/points" />
    </node>

</launch>
